static/usr/lib/udev/rules.d/90-rtc-sys-time-init.rules

```diff
@@ -0,0 +1,12 @@
+# When device rtc* is added
+# (1) and the RTC module have successfully set system time from RTC (indicated with hctosys attribute)
+# (2) and the kernel command line does not contain parameter rtc_sys_time_init that will cause
+#     systemd generator rtc-sys-time-init-generator to instantiate rtc-sys-time-init@.service
+# instantiate rtc-sys-time-init@.service for the device node
+IMPORT{cmdline}="rtc_sys_time_init"
+ACTION=="add", SUBSYSTEM=="rtc", KERNEL=="rtc*", ATTR{hctosys}=="1", ENV{rtc_sys_time_init}!="?*", TAG+="systemd", PROGRAM="/usr/bin/systemd-escape --path %N", ENV{SYSTEMD_WANTS}+="rtc-sys-time-init@%c.service"
```

Comment:
I think we are missing some `IMPORT{cmdline}` to read `rtc_sys_time_init`.

Reply:
Ah it is before. Though I would have but it after `ATTR{hctosys}=="1"`, but I think it is fine like that.

Reply:
The IMPORT only becomes available to rules after the rule that imports. I have read this, and tested this, having it in the same line does not work.